Forms/MainForm.cs

```diff
@@ -87,7 +87,7 @@ private void InitializeTooltips()
             TooltipBuilder.SetTooltip(cFloors, "Select a floortype for every floor ingame:\n\n - Default: Vanilla floortypes.\n - Sand: Link sinks slowly into every floor, affecting movement speed.\n - Ice: Every floor is slippery.\n - Snow: Similar to sand. \n - Random: Any random floortypes of the above.");
             TooltipBuilder.SetTooltip(cClockSpeed, "Modify the speed of time.");
             TooltipBuilder.SetTooltip(cHideClock, "Clock UI will be hidden.");
-            TooltipBuilder.SetTooltip(cNoStartingItems, "You will not start with any randomized starting items.");
+            TooltipBuilder.SetTooltip(cRandomStartingItems, "You will start with any randomized items obtained during first cycle.\nSuch as Sword, Shield, Heart Containers, Song of Healing and Deku Mask.");
```

Comment:
Wording is a bit weird. Sword, Shield, and Heart Containers aren't really "obtained" during first cycle.

Reply:
I added, "Obtained before or during first cycle". The list should also help clear that up.